lib/diagnostic.rb

```diff
@@ -161,3 +182,5 @@ def multiplier
     end
   end
 end
+
```

Comment:
In direct inheritance, you're limited to inheriting from a single source.
In composition, you can draw from as many modules as you like.